### PR TITLE
fixes adamantine armor slowdown

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
adamantine armor now works as intended, and the slowdown cannot be removed

## Why It's Good For The Game
this shit is waaaaay too easy to cheese

## Changelog
:cl:
fix: adamantine armor can no longer be cheesed with things that undo armor slowdown like morphine or red slime
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Would this not be a core underlying issue with all armour then?

**Comment by zeskorion** [comment]
> Would this not be a core underlying issue with all armour then?

no. Adamantine armor is explicitly not supposed to be sped up, whether via speed potions, morphine, stable slime cores, etc, which can be done right now

**Comment by Firefox13** [comment]
изображение
You forgot a dot at the end.

File: code/modules/research/xenobiology/crossbreeding/_clothing.dm [hunk 1]
```diff
@@ -129,14 +129,21 @@ Slimecrossing Armor
 
 /obj/item/clothing/suit/armor/heavy/adamantine
 	name = "adamantine armor"
-	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do."
+	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do. The locked joints appear to move under their own power, making this suit of armor impossible to speed up"
```
Comment (ike709) [inline code/modules/research/xenobiology/crossbreeding/_clothing.dm]:
```suggestion
	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do. The locked joints appear to move under their own power, making this suit of armor impossible to speed up."
```

**Pull request merged**